great-idea-website/css/index.css

```diff
@@ -63,4 +63,100 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+.container {
+	margin: 50px 100px;
+	position: relative;
+}
+
+.logo {
+	position: absolute;
+	top: -15px;
+	left: 75%;
+}
+
+nav > a {
+	left: 0;
+	padding: 50px 85px 50px 0px;
+	font-size: 1.3rem;
+	text-decoration: none;
+	color: grey;
+}
+
+nav > a:hover {
+	color: black;
+}
+
+.big-text {
+	display: inline-block;
+	margin: 120px 30px 50px 250px;
+	width: 300px;
+	height: 400px;
+	font-size: 5rem;
+	text-align: center;
+}
+
+button {
+	width: 250px;
+	height: 50px;
+	font-size: 1.2rem;
+	background: white;
+	border: 1px solid black;
+}
+
+button:hover {
+	background-color: black;
+	color: white;
+}
+
+.header-img {
+	position: absolute;
+	margin: 100px 0px 50px 100px;
+}
+
+
+.top-section > div {
+	padding: 30px 30px 30px 0;
+}
+
+.left-div {
+	float: left;
+	width: 50%;
+}
+
+.right-div {
+	float: right;
+	width: 50%;
+}
+
+.middle-img {
+	width: 100%;
+}
+
+.bottom-section > div {
+	padding: 30px 30px 30px 0;
+}
+
+.first-div {
```

Comment:
It's generally better to avoid class naming with first, second, etc. and to instead choose a class name that describes the content. Also, since these three all have the same styles, one class could've been applied to each of them.